scripts/oncall-email/curl.shell

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+script='query {
+  organization(login:\"returntocorp\") {
+    project(number:50) {
+      columns(first:10) {
+       	nodes {
+          name
+       	  cards {
+       	    nodes {
+              content{
+                __typename
+                ...on Issue{
+                  title
+                  url
+                  assignees(first:3) {
+                     nodes {
+                      login
+                    }
+                  }
+                  closed
+                  createdAt
+                  updatedAt
+                  labels(first:5) {
+                    nodes{
+                      name
+                    }
+                  }
+                }
+              }
+              isArchived
+              note
+              creator {
+                login
+              }
+       	    }
+       	  }
+       	}
+      }
+	  }
+  }
+}'
+script="$(echo $script)"
```

Comment:
Please explain what this script is for, for the sake of everyone who will stumble upon it.

You should also explain the intention of the line `script="$(echo $script)"`, which appears to be replacing whitespace by single spaces.

See https://github.com/mjambon/reasonable-shell-script for more guidelines on writing shell scripts.